orchestra/static/orchestra/task/js/directives.js

```diff
@@ -16,6 +16,22 @@
     };
   });
 
+  angular.module('orchestra.task.directives').directive(
```

Comment:
curiously should anything else use this directive?  trying to understand where it fits in